scripts/configure_nexus.sh

```diff
@@ -62,67 +62,26 @@ if [ -z "$NEXUS_PASS" ]; then
     export NEXUS_PASS=$NEW_PASSWORD
```

Comment:
Not the perfect place to put this comment in:
Seems like we have a lot of files directly related to Nexus at the root of the project folder structure. What do you think about moving all of them into the nexus template folder @oliver7598 ?

Reply:
I tend to agree. Although might be that's taken as a separate issue that includes building these steps into the nexus bundle?

Reply:
I'm fine with waiting as long as we open an issue and reference it here for the comment to be resolved. 
Although, I don't see any knock-on affect with regards to the nexus things (aad app reg is outside the scope). If this is going to impact github actions and/or AzDo pipelines something else is wrong as we shouldn't directly use any of this...

To clarify, we're talking about moving the configure_nexus.sh script and the nexus_config folder into the template folder.

Reply:
Nexus config scripts and related files have been moved into the nexus shared-service folder.  

Docs also updated to reflect new script path in configure shared services section